### ISSUE TITLE
windows 10 minikube start hangs after "Creating hyperv VM" message 

### ISSUE THREAD

**Issue by GeorgeLittleDrake** [docker/for-win#4721]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 16345F8C-52BE-4B69-B158-65AE06239BE4/20190914122339

### Expected behavior
I'd like to process article "Getting started with Kubernetes" (Java Magazine May/June 2019)
### Actual behavior
In order to start minikube I have downloaded and installed minikube (1.1.1) using windows installer (according to https://kubernetes.io/docs/tasks/tools/install-minikube/). The first time everything seemed to be ok, however followed by prompts I installed minikube 1.3.1.

Now it doesn't matter which version I run, the command prompt freezes.
First steps I made was commands:
Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All
New-VMSwitch -name ExternalSwitch -NetAdapterName Ethernet -AllowManagementOS $true
minikube config set hyperv-virtual-switch ExternalSwitch
minikube config set vm-driver hyperv

after 
minikube start
the result is:
<result>
* minikube v1.3.1 on Microsoft Windows 10 Pro 10.0.17763 Build 17763
* Downloading VM boot image ...
minikube-v1.3.0.iso.sha256: 65 B / 65 B [--------------------] 100.00% ? p/s 0s
minikube-v1.3.0.iso: 131.07 MiB / 131.07 MiB [-------] 100.00% 2.23 MiB p/s 59s
* Creating hyperv VM (CPUs=2, Memory=4096MB, Disk=20000MB) ...

If I don't remove .minicube directory the result is similar despite the downloading info.
</result>
### Information

  - Windows Version: Windows 10 pro, windows version 1809
  - Docker Desktop Version: 2.1.0.2 stable (37877)
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: No

### Steps to reproduce the behavior

To reproduce the error it is enough:
  1. start minikube

**Comment by thaJeztah** [comment]
Might be better to ask for support from the minikube maintainers.

Wondering though why you're installing minikube? Docker Desktop comes with Kubernetes out of the box; you can enable it in the configuration panel, and you don't need minikube for that

**Comment by GeorgeLittleDrake** [comment]
Thank you for your answer. You are definitely right that I don't need minikube - but I didn't know this. Probably I misunderstood the articles I'd been reading (mentioned in my previous post). I'll try to work according to your comment and utilize kubernets tools supplied together with Docker Desktop.
On the other hand, I suppose I solved my problem with minikube:
I started with minikube at work, where I'm connected to the network via ethernet card - so I connected a virtual switch to this adapter - it worked fine. I brought my laptop home and tried to run minikube based on an out-of-date network adapter. Now  I created new virtual switch which I connected to wifi adapter and I can start:

<result>
C:\WINDOWS\system32>minikube start --vm-driver=hyperv --hyperv-virtual-switch=VirtualSwitch

&ast; minikube v1.3.1 on Microsoft Windows 10 Pro 10.0.17763 Build 17763

&ast; Downloading VM boot image ...
minikube-v1.3.0.iso.sha256: 65 B / 65 B [--------------------] 100.00% ? p/s 0s
minikube-v1.3.0.iso: 131.07 MiB / 131.07 MiB [------] 100.00% 2.16 MiB p/s 1m1s
&ast; Creating hyperv VM (CPUs=2, Memory=4096MB, Disk=20000MB) ...

&ast; Preparing Kubernetes v1.15.2 on Docker 18.09.8 ...

&ast; Downloading kubeadm v1.15.2

&ast; Downloading kubelet v1.15.2

&ast; Pulling images ...

&ast; Launching Kubernetes ...

&ast; Waiting for: apiserver proxy etcd scheduler controller dns

&ast; Done! kubectl is now configured to use "minikube"
</result>
